storybook/src/components/TextInput/TextInput.docs.mdx

```diff
@@ -14,14 +14,116 @@ import README from "../../../../packages/css/src/components/text-input/README.md
 
 ## Examples
```

Comment:
Should we link to NLDS and gov.uk in these docs for more info? For example [the NLDS docs on placeholders](https://www.nldesignsystem.nl/richtlijnen/formulieren/placeholders), or the gov.uk docs on [passwords](https://design-system.service.gov.uk/patterns/passwords/) or [phone numbers](https://design-system.service.gov.uk/patterns/telephone-numbers/).

Reply:
Yes! I’ve added them and shortened the docs I had.